packages/carbon-graphs/README.md

```diff
@@ -86,25 +86,25 @@ The data can now by plotted by as a line graph by using the following:
 
 This results in the following graph:
 
-![Alt](../terra-docs/docs/assets/carbon-simple-line-graph.png "simple line graph")
+![Alt](../terra-graphs-docs/docs/assets/carbon-simple-line-graph.png "simple line graph")
 
 
 ## Graph types
 
 The following graph types can be plotted using Carbon:
 
--   [Line](../terra-docs/docs/controls/Line.md#usage)
--   [Multi Line](../terra-docs/docs/controls/Line.md#multi-line)
--   [Spline Line](../terra-docs/docs/controls/Line.md#spline-line)
--   [Paired Result](../terra-docs/docs/controls/PairedResult.md#usage)
--   [Bar](../terra-docs/docs/controls/Bar.md#usage)
--   [Timeline](../terra-docs/docs/controls/Timeline.md#usage)
--   [Pie](../terra-docs/docs/controls/Pie.md#usage)
--   [Gantt](../terra-docs/docs/controls/Gantt.md#usage)
--   [Scatter](../terra-docs/docs/controls/Scatter.md#usage)
--   [Bubble](../terra-docs/docs/controls/Bubble.md#usage)
-    -   [Bubble Single Dataset](../terra-docs/docs/controls/BubbleSingleDataset.md)
-    -   [Bubble Multiple Dataset](../terra-docs/docs/controls/BubbleMultipleDataset.md)
+-   [Line](../terra-graphs-docs/docs/controls/Line.md#usage)
```

Comment:
Looks like these links are not working as expected. 
![image](https://user-images.githubusercontent.com/63670637/128206141-aa9f5713-1509-48c8-99de-eb97330b1d43.png)

Reply:
They should be working on GitHub. If you select `Rich Text Difference` and try the links, they should work.

![CleanShot 2021-08-04 at 20 18 08@2x](https://user-images.githubusercontent.com/38024451/128276132-a1abddf9-6525-4e42-a321-8de662f51813.png)

Reply:
Even in local, it dosen't look good
![image](https://user-images.githubusercontent.com/63670637/128568075-ce973f3f-1342-493d-9258-50f638dcf9c6.png)

Reply:
This is also happening in other .md files that weren't updated so I think it's a rendering issue. Either way, it's beyond the scope of this story.